packages/atomic/src/components/common/interface/i18n.ts

```diff
@@ -61,5 +61,12 @@ export function initi18n(atomicInterface: BaseAtomicInterface<AnyEngineType>) {
     interpolation: {
       escape: (str) => DOMPurify.sanitize(str),
     },
+    compatibilityJSON: 'v3',
   });
 }
+
+declare module 'i18next' {
+  interface CustomTypeOptions {
+    resources: Record<string, Record<string, string>>;
```

Comment:
Where is this type used ?

Reply:
It's used by `i18n.t(...)`.

Reply:
So, is it really useful then ? Or mostly an artefact of trying to add typing to the available keys ?

Reply:
Yeah, we need it to compile, since i18next now tries to add typings by default.